riotfile.py

```diff
@@ -2351,12 +2351,11 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         Venv(
             name="opentelemetry",
             command="pytest {cmdargs} tests/opentelemetry",
-            # FIXME: this test suite breaks on 3.7
             pys=select_pys(min_version="3.8"),
             pkgs={
                 "pytest-randomly": latest,
                 "pytest-asyncio": "==0.21.1",
-                "opentelemetry-api": ["~=1.0.0", "~=1.3.0", "~=1.4.0", "~=1.8.0", "~=1.11.0", "~=1.15.0", latest],
+                "opentelemetry-api": ["~=1.0.0", "~=1.15.0", latest],
```

Comment:
nit: Should we pin on 1.25.0 explicitly and latest? Just so we know the "latest" that should have worked in case the next latest also is incompatible? 
```suggestion
                "opentelemetry-api": ["~=1.0.0", "~=1.15.0", "~=1.25.0", latest],
```

Reply:
I made this change in the next PR: https://github.com/DataDog/dd-trace-py/pull/9941/files#diff-8c2dff2cbcc0f4fff63050381d4c7546ae7195383ec8f9cc993f8494b77a34c3. We actually need to test 1.26.0 and not just 1.25.0. Testing both is a bit overkill.